defold-rive/editor/src/rive.clj

```diff
@@ -506,10 +511,13 @@
       (restore-gl gl))))
 
 (defn- render-rive-scenes [^GL2 gl render-args renderables rcount]
-  (let [pass (:pass render-args)]
+  (let [pass (:pass render-args)
+        render-groups (collect-render-groups renderables)]
+       (doseq [group render-groups]
+         (plugin-update-file (:handle group) 0.0 (:texture-set-pb group)))
```

Comment:
This seems very expensive to do on every render call. Shouldn't we try to cache it as an output on the `RiveModelNode`?

Reply:
Yeah, sounds good 👍